Formula/cgal.rb

```diff
@@ -11,57 +11,28 @@ class Cgal < Formula
     sha256 "861fcbdd2cea9fd8365e53e8ec7218f11c33b6b2fc1d0d732eeaa39c1b0343fd" => :yosemite
   end
 
-  option :cxx11
```

Comment:
Let's not remove this until we have a general solution to the C++11 situation on <= 10.9.